### ISSUE TITLE
Deprecated event

### ISSUE THREAD

**Issue by SlimeDog** [RockinChaos/ItemJoin#125]
Spigot 1.12 latest (today)
ItemJoin 4.2

```
[09:14:18] [Server thread/WARN]: [ItemJoin] "ItemJoin v4.2" has registered a listener for org.bukkit.event.player.PlayerPickupItemEvent on method "public void me.RockinChaos.itemjoin.listeners.Pickups.onPickup(org.bukkit.event.player.PlayerPickupItemEvent)", but the event is Deprecated. "Server performance will be affected"; please notify the authors [RockinChaos].
```

**Comment by SlimeDog** [comment]
Spigot 1.12 latest introduced EntityPickupItemEvent, and eventually it will be required; event PlayerPickupItemEvent is deprecated.

**Comment by RockinChaos** [comment]
Interesting, I will look into this. Thanks for bring it to my attention :)

**Comment by RockinChaos** [comment]
Any reason I cant find this? I tried a fresh compile of spigot / craftbukkit and it says that the EntityPickupItemEvent does not exist and I can't find it on spigots commit list.

**Comment by RockinChaos** [comment]
Feature has been added and released in v4.3
You can get it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=176180
